src/main/scala/scalismo/common/interpolation/BarycentricInterpolator.scala

```diff
@@ -33,48 +33,33 @@ object BarycentricInterpolator {
 case class BarycentricInterpolator3D[A: ValueInterpolator](mesh: TetrahedralMesh[_3D])
     extends BarycentricInterpolator[_3D, A] {
 
-  override protected val valueInterpolator: ValueInterpolator[A] = ValueInterpolator[A]
-
-  // TODO: Temporary solution, replace for Milestone M2!
-  private def getTetrahedralMeshCell(p: Point[_3D]): Option[TetrahedralCell] = {
-
-    val numberOfTetrahedrons = mesh.tetrahedralization.tetrahedrons.length
-
-    def isInsideCell(tc: TetrahedralCell): Boolean = mesh.isInsideTetrahedralCell(p, tc)
-    val isInsideCellMemoized = Memoize(isInsideCell, numberOfTetrahedrons)
-
-    var cell: Option[TetrahedralCell] = None
-    var neighbourhood = Set[TetrahedronId]()
+  val meshOps = MeshOperations(mesh)
```

Comment:
Is there a reason why not to use `mesh.operations`?

Reply:
Not really, will change this.